Update 2024-08-28-community-meeting.mdx (#583)
wasmCloud/wasmCloud.com: community/2024-08-28-community-meeting.mdx

### DEMO: wasmCloud 1.2 / wash 0.31 showcase

- wasmCloud 1.2 is our latest release, now out! Check out this a great blog post on [wasmCloud 1.2](https://wasmcloud.com/blog/2024-08-27-wasmcloud-1.2-improved-streams-automatic-config-updates) which is well worth diving into.
- Highlights include: automatic config updates for providers (update to the latest SDK - there is a great first issue for implementing this in Go), improved streams, additional policy and secrets validation, better downstream link validation, and fully-tunable wasmtime runtime parameters (important to be able to control how much memory a component can use).
- In focus: `wasi-blobstore` streaming has seen a whole suite of improvements; manifests are better. Suggest checking out the [blobby example](https://github.com/wasmCloud/wasmCloud/blob/04d7e8e902ec27dcc20e1deeb222574554166ad1/examples/rust/components/blobby/README.md?plain=1#L4) to see how this works in practice.
- This closes out all the remaining issues in the Q2 roadmap.
- In wash 0.31 we have a couple of great improvements - as always check out the recording for Brooks's demo.

> - FEATURE: If you output the version for wash we now get the version and the embeded default versions of NATS, wadm and wasmCloud that wash will utilize. This is super useful in the command line - easy to do wash version and get all the versions needed by wash.

Reviewer: ```suggestion
- FEATURE: If you output the version for wash you now also get the versions of NATS, wadm and wasmCloud that wash will utilize. This is super useful in the command line - easy to run `wash --version` and get the versions for the entire ecosystem.
```

> - FEATURE: improvements to the tables that we output in the command line. wash app lists - instead of having many separate fields, we have the description listed below the app and have streamlined the information there.

Reviewer: ```suggestion
- FEATURE: improvements to the tables that we output in the command line. `wash app list` - instead of having many separate fields, we have the description listed below the app and have streamlined the information there.
```

- We have also updated the way we output to the terminal so we don't exceed the width of the terminal, which helps wth consistency.

> - FEATURE: Thank you for your work [Aditya Salunkhe](https://github.com/Aditya1404Sal) in enabling us to watch the status of wash app lists - instant refresh allows us to reconcile before compiling.

Reviewer: ```suggestion
- FEATURE: Thank you for your work [Aditya Salunkhe](https://github.com/Aditya1404Sal) in adding a `--watch` argument to `wash app list` - instant refresh allows us to reconcile before compiling.
```

> - FEATURE: We have a much improved wash status view. We can see a lot more information about the application and its inner state. When we query the wash app status, we can now see why it might not reconcile - infinitely more helpful.

Reviewer: ```suggestion
- FEATURE: We have a much improved `wash app status` view. We can see a lot more information about the application and its inner state. When we run `wash app status`, we can now see why it might not reconcile - infinitely more helpful.
```
